zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -337,7 +337,6 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
-
```

Comment:
Sorry for this noise. I was using empty lines to track my progress. I can put these back if you'd prefer.

Reply:
All good 👍